Azure/sap-hana
# Add webdispatcher module

## Problem:
No webdispatcher at the moment.

## Solution:
Add code to deploy webdispatcher in app tier.

## Test:
1. Deploy with below definition in input JSON and finished fine:
```
"application": {
    "sid": "HN1",
    "enable_deployment": true,
    "scs_instance_number": "01",
    "ers_instance_number": "02",
    "scs_high_availability": false,
    "application_server_count": 1,
    "webdispatcher_count": 2,
    "authentication": {
      "type": "key",
      "username": "azureadm"
    },
    "os": {
      "publisher": "suse",
      "offer": "sles-sap-12-sp5",
      "sku": "gen1"
    }
  },
```
1. Change the number of `webdispatcher_count`, see inventory ini/yml changed accordingly locally and remotely on RTI
